index.php

```diff
@@ -23,7 +23,8 @@
 <a href="http://github.com/lmanul/whylinuxisbetter">GitHub</a> and come contribute!</p>
 
 <p>Have a look at <a href="switch/try/oneday/index.php">One day with Linux (a screenshot tour)</a>!</p>
-
+<p>Note: The following features are not specific to just Linux, 

```

Comment:
I don't understand what warrants a mention to BSDs (especially at the top of the front page) when the site is specifically about Linux. Sounds like a desperate "hey, I exist too, look at meeeee" call :-)